skbio/sequence/_grammared_sequence.py

```diff
@@ -450,6 +466,32 @@ def has_degenerates(self):
         return bool(self.degenerates().any())
 
     @stable(as_of='0.4.0')
```

Comment:
This should be `0.4.2-dev` since the new name will be stable in the next release.